<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetChatInput</class>
 <widget class="QMainWindow" name="WidgetChatInput">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>97</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>97</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chat Text Input</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Library/Document.png</normaloff>:/Resource/Library/Document.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonSend">
         <property name="text">
          <string>Send</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Chat/Send.png</normaloff>:/Resource/Chat/Send.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="helpLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CWidgetReturnEmitTextEdit" name="textEditInput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarTextTools">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
  </widget>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/FormatTextBold.png</normaloff>:/Resource/Generic/FormatTextBold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/FormatTextItalic.png</normaloff>:/Resource/Generic/FormatTextItalic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/FormatTextUnderline.png</normaloff>:/Resource/Generic/FormatTextUnderline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWidgetReturnEmitTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widgetreturnemittextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
